provisioner/ansible/provisioner.go

```diff
@@ -382,6 +391,21 @@ func validateFileConfig(name string, config string, req bool) error {
 	return nil
 }
 
+func validateDirectoryConfig(name string, config string, req bool) error {
```

Comment:
Ok, I hardcoded `inventory_directory` and changed the name of the function to `validateInventoryDirectoryConfig`.